.github/workflows/cml-with-dvc.yaml

```diff
@@ -19,25 +18,31 @@ jobs:
         run: |
           cml runner \
               --cloud=aws \
-              --labels=cml-runner \
               --cloud-region=eu-west \
               --cloud-type=t2.micro \
+              --labels=cml-runner \
               --single
   train-model:
     needs: deploy-runner
     runs-on: [self-hosted, cml-runner]
     timeout-minutes: 120 # 2h
-    container:
-      image: docker://iterativeai/cml:0-dvc2-base1
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - uses: iterative/setup-cml@v1
       - name: Train model
         env:
           REPO_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
-          GDRIVE_CREDENTIALS_DATA: ${{ secrets.GDRIVE_CREDENTIALS_DATA }}
```

Comment:
not needed since you're injecting into `creds.json` later directly from `secrets` (ignoring env var)

Reply:
The `creds.json` part was a workaround pending [this change](https://github.com/iterative/dvc/pull/7213#event-6398807185). I think it'd be cleaner to rely only on env vars now that the fix has been applied. Still need to test it, though.